sentry/src/main/java/io/sentry/Session.java

```diff
@@ -79,7 +78,7 @@ public Session(
       final @Nullable Date timestamp,
       final int errorCount,
       final @Nullable String distinctId,
-      final @Nullable UUID sessionId,
+      final @Nullable String sessionId,
```

Comment:
do we have to apply some validation still that this is compliant with the uuid format?